addons/im_livechat/static/src/models/public_livechat_window.js

```diff
@@ -49,5 +46,8 @@ registerModel({
             inverse: 'publicLivechatWindowOwner',
             isCausal: true,
         }),
+        widget: attr({
+            default: null,
```

Comment:
why null instead of undefined?

Reply:
Was set to `null` before refactoring, so just dummy keeping this.

It looks useless now that the widget is managed by model itself, so it's not possible to have `.widget` being falsy actually (except in small critical sections when computing fields, but it's never guarded).

So will remove this `default: null` in this PR